### PR TITLE
June 11th Launch

### PR DESCRIPTION
**Pull request by stephanie-anderson**
Please point and merge all PRs for the June 11th launch into this branch

### CODE REVIEW

**Review by Jeffreyhung — APPROVED** [review]

File: docs/concepts/key-terms/key-terms.mdx [hunk 1]
```diff
@@ -50,11 +50,11 @@ Each of these key features in [sentry.io](https://sentry.io) is represented by a
...
 
 - **Issues** - Displays information about grouped problems in your application. From here, you can go to the **Issue Details** page for a more granular view of any issue. Learn more in the full [Issues documentation](/product/issues/).
 
-- **Performance** - The main view in [sentry.io](http://sentry.io) where you can search or browse for transaction data. The page displays graphs that visualize transactions or trends, as well as a table where you can view relevant transactions and drill down to more information about them. Learn more in the full [Performance documentation](/product/performance/).
+- **Performance** - The main view in [sentry.io](http://sentry.io) where you can search or browse for transaction data. The page displays graphs that visualize transactions or trends, as well as a table where you can view relevant transactions and drill down to more information about them. Learn more in the full [Tracing documentation](/product/performance/).
```
Comment (lizokm) [inline docs/concepts/key-terms/key-terms.mdx:57]:
```suggestion
- **Performance** - A view in [sentry.io](http://sentry.io) where you can search or browse for transaction data. The page displays graphs that visualize transactions or trends, as well as a table where you can view relevant transactions and drill down to more information about them. Learn more in the full [Performance monitoring documentation](/product/performance/).
```

File: docs/contributing/approach/product-docs/write-index.mdx [hunk 2]
```diff
@@ -41,7 +41,7 @@ Use this approach for a feature that doesn’t require much explanation and has
 
 ## How: Wider-Ranging Features - Benefit Breakdown
 
-Use this approach for a page that requires more explanation to break down and/or has a lot of child pages to explain its different benefits, like [Performance Monitoring](/product/performance/).
+Use this approach for a page that requires more explanation to break down and/or has a lot of child pages to explain its different benefits, like [Tracing](/product/performance/).
```
Comment (lizokm) [inline docs/contributing/approach/product-docs/write-index.mdx:44]:
```suggestion
Use this approach for a page that requires more explanation to break down and/or has a lot of child pages to explain its different benefits, like [Performance monitoring](/product/performance/).
```

File: docs/platforms/android/tracing/index.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+title: Set Up Tracing
+description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."
```
Comment (lizokm) [inline docs/platforms/android/tracing/index.mdx:3]:
```suggestion
description: "Learn how to enable tracing in your app so you can get valuable performance insights about your application."
```

File: docs/platforms/android/tracing/instrumentation/custom-instrumentation.mdx [hunk 4]
```diff
@@ -1,12 +1,12 @@
 ---
 title: Custom Instrumentation
-description: "Learn how to capture performance data on any action in your app."
 sidebar_order: 20
+description: "Learn how to capture performance data on any action in your app."
```
Comment (lizokm) [inline docs/platforms/android/tracing/instrumentation/custom-instrumentation.mdx:4]:
```suggestion
description: "Learn how to enable trace and span data-capturing on any action in your app."
```

File: docs/platforms/apple/common/tracing/index.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+title: Set Up Tracing
+description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."
```
Comment (lizokm) [inline docs/platforms/apple/common/tracing/index.mdx:3]:
```suggestion
description: "Learn how to enable tracing in your app and get valuable performance insights about your application."
```

File: docs/platforms/dart/tracing/index.mdx [hunk 6]
```diff
@@ -0,0 +1,38 @@
+---
+title: Set Up Tracing
+description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."
```
Comment (lizokm) [inline docs/platforms/dart/tracing/index.mdx:3]:
```suggestion
description: "Learn how to enable tracing in your app and get valuable performance insights about your application."
```

File: docs/platforms/dotnet/common/migration/index.mdx [hunk 7]
```diff
@@ -121,7 +121,7 @@ This SDK version is compatible with a self-hosted version of Sentry `22.12.0` or
   - `Runtime.Clone()` - this shouldn't have been public in the past and has no replacement.
   - `AssemblyExtensions` - this shouldn't have been public in the past and has no replacement.
   - `SentryDatabaseLogging.UseBreadcrumbs()` - it is called automatically and has no replacement.
-- A number of interfaces have removed:  
+- A number of interfaces have removed:
```
Comment (lizokm) [inline docs/platforms/dotnet/common/migration/index.mdx:124]:
```suggestion
- A number of interfaces have been removed:
```

File: docs/platforms/dotnet/common/migration/index.mdx [hunk 8]
```diff
@@ -248,7 +248,7 @@ The replacement is the option `MaxRequestBodySize`.
 
 #### New Methods on `IHub` and `ISentryClient`
 
-Sentry's performance monitoring capabilities requires additive API changes to interfaces such as `IHub` and `ISentryClient`. If you implement your own `IHub`, you'll need to implement two new methods:
+Sentry's tracing capabilities requires additive API changes to interfaces such as `IHub` and `ISentryClient`. If you implement your own `IHub`, you'll need to implement two new methods:
```
Comment (lizokm) [inline docs/platforms/dotnet/common/migration/index.mdx:251]:
```suggestion
Sentry's tracing capabilities require additive API changes to interfaces such as `IHub` and `ISentryClient`. If you implement your own `IHub`, you'll need to implement two new methods:
```

File: docs/platforms/dotnet/common/tracing/index.mdx [hunk 9]
```diff
@@ -0,0 +1,42 @@
+---
+title: Set Up Tracing
+description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."
```
Comment (lizokm) [inline docs/platforms/dotnet/common/tracing/index.mdx:3]:
```suggestion
description: "Learn how to enable tracing in your app and get valuable performance insights about your application."
```

File: docs/platforms/flutter/tracing/index.mdx [hunk 10]
```diff
@@ -0,0 +1,38 @@
+---
+title: Set Up Tracing
+description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."
```
Comment (lizokm) [inline docs/platforms/flutter/tracing/index.mdx:3]:
```suggestion
description: "Learn how to enable tracing in your app and get valuable performance insights about your application."
```

File: docs/platforms/java/common/tracing/index.mdx [hunk 11]
```diff
@@ -0,0 +1,64 @@
+---
+title: Set Up Tracing
+description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."
```
Comment (lizokm) [inline docs/platforms/java/common/tracing/index.mdx:3]:
```suggestion
description: "Learn how to enable tracing in your Java SDK and get valuable performance insights about your application."
```

File: docs/platforms/java/common/tracing/instrumentation/apollo.mdx [hunk 12]
```diff
@@ -2,7 +2,7 @@
 title: Apollo Integration
 sidebar_order: 30
 sdk: sentry.java.apollo
-description: "Learn how to capture the performance of Apollo GraphQL client."
+description: "Learn how to capture tracing information of the Apollo GraphQL client."
```
Comment (lizokm) [inline docs/platforms/java/common/tracing/instrumentation/apollo.mdx:5]:
```suggestion
description: "Learn how to capture tracing information from the Apollo GraphQL client."
```

File: docs/platforms/java/common/tracing/instrumentation/apollo3.mdx [hunk 13]
```diff
@@ -2,7 +2,7 @@
 title: Apollo3 Integration
 sidebar_order: 31
 sdk: sentry.java.apollo-3
-description: "Learn how to capture the performance of Apollo GraphQL client."
+description: "Learn how to capture tracing information of the Apollo GraphQL client."
```
Comment (lizokm) [inline docs/platforms/java/common/tracing/instrumentation/apollo3.mdx:5]:
```suggestion
description: "Learn how to capture tracing information from the Apollo GraphQL client."
```

File: docs/platforms/java/common/tracing/instrumentation/file-io.mdx [hunk 14]
```diff
@@ -1,7 +1,7 @@
 ---
 title: File I/O Integration
 sidebar_order: 60
-description: "Learn how to capture the performance of file I/O operations."
+description: "Learn how to capture tracing information for file I/O operations."
```
Comment (lizokm) [inline docs/platforms/java/common/tracing/instrumentation/file-io.mdx:4]:
```suggestion
description: "Learn how to capture tracing information from file I/O operations."
```

File: docs/platforms/java/common/tracing/instrumentation/jdbc.mdx [hunk 15]
```diff
@@ -2,7 +2,7 @@
 title: JDBC Instrumentation
 sidebar_order: 40
 sdk: sentry.java.jdbc
-description: "Learn how to capture the performance of database queries executed with JDBC."
+description: "Learn how to capture tracing information for database queries executed with JDBC."
```
Comment (lizokm) [inline docs/platforms/java/common/tracing/instrumentation/jdbc.mdx:5]:
```suggestion
description: "Learn how to capture tracing information from database queries executed with JDBC."
```

File: docs/platforms/java/common/tracing/instrumentation/okhttp.mdx [hunk 16]
```diff
@@ -2,7 +2,7 @@
 title: OkHttp Integration
 sidebar_order: 35
 sdk: sentry.java.okhttp
-description: "Learn how to capture the performance of OkHttp client."
+description: "Learn how to capture tracing information of the OkHttp client."
```
Comment (lizokm) [inline docs/platforms/java/common/tracing/instrumentation/okhttp.mdx:5]:
```suggestion
description: "Learn how to capture tracing information from the OkHttp client."
```

File: docs/platforms/native/common/tracing/index.mdx [hunk 17]
```diff
@@ -0,0 +1,53 @@
+---
+title: Set Up Tracing
+description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."
```
Comment (lizokm) [inline docs/platforms/native/common/tracing/index.mdx:3]:
```suggestion
description: "Learn how to enable tracing in your Native SDK and get valuable performance insights about your application."
```

File: docs/platforms/php/common/tracing/index.mdx [hunk 18]
```diff
@@ -1,20 +1,20 @@
 ---
-title: Set Up Performance
-description: "Learn how to enable performance monitoring in your app if it is not already set up."
+title: Set Up Tracing
+description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."
```
Comment (lizokm) [inline docs/platforms/php/common/tracing/index.mdx:3]:
```suggestion
description: "Learn how to enable tracing in your PHP SDK and get valuable performance insights about your application."
```

File: docs/platforms/powershell/tracing/index.mdx [hunk 19]
```diff
@@ -0,0 +1,36 @@
+---
+title: Set Up Tracing
+description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."
```
Comment (lizokm) [inline docs/platforms/powershell/tracing/index.mdx:3]:
```suggestion
description: "Learn how to enable tracing in your PowerShell SDK and get valuable performance insights about your application."
```

File: docs/platforms/python/tracing/index.mdx [hunk 20]
```diff
@@ -0,0 +1,25 @@
+---
+title: Set Up Tracing
+description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."
```
Comment (lizokm) [inline docs/platforms/python/tracing/index.mdx:3]:
```suggestion
description: "Learn how to enable tracing in your Python SDK and get valuable performance insights about your application."
```

**Review by lizokm — APPROVED** [review]
Thank you SO SO MUCH for putting this huge PR together. I have a few small wording suggestions. One thing to note - we do still have the Performance Monitoring page, so I made a few tweaks at the beginning of the PR to keep referring to that very specific page as "Performance Monitoring" instead of "tracing". Otherwise, this looks great!

**Comment by a-hariti** [comment]
I did a master branch merge and defaulted to this branch on conflicting files where resolution was not immediately obvious (`git checkout --ours $file` during the merge), specifically:

```
docs/platforms/java/common/metrics/index.mdx
docs/platforms/javascript/common/configuration/integrations/browserprofiling.mdx
docs/platforms/javascript/guides/aws-lambda/index.mdx
docs/platforms/javascript/guides/gcp-functions/index.mdx
docs/product/insights/web-vitals/index.mdx
platform-includes/distributed-tracing/how-to-use/javascript.mdx
platform-includes/distributed-tracing/how-to-use/javascript.nextjs.mdx
platform-includes/distributed-tracing/how-to-use/javascript.node.mdx
platform-includes/distributed-tracing/how-to-use/javascript.remix.mdx
platform-includes/distributed-tracing/how-to-use/javascript.sveltekit.mdx
```

Authors of these files please double check things 🙏

**Comment by a-hariti** [comment]
I see a failing build due with this error: `missing parent and grandparent: /some/long/path`

This generally means missing `index.mdx` files on segments of said path (ie no `/some/long/index.mdx` in the above)

File: docs/platforms/dotnet/index.mdx [hunk 21]
```diff
@@ -0,0 +1,97 @@
...
+## Configure
+
+To capture all errors, even the one during the startup of your application, you should initialize the Sentry .NET SDK as soon as possible.
+
+<SignInNote />
+
+```csharp {"onboardingOptions": {"performance": "6-7", "profiling": "8-9"}}
+SentrySdk.Init(options =>
+{
+    options.Dsn = "https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.us.sentry.io/5428537";
```
Comment (jamescrosswell) [inline docs/platforms/dotnet/index.mdx:72]: Is this meant to be a hard coded DSN?

**Pull request merged**